From: Michael Biebl <biebl@debian.org>
Date: Fri, 28 Feb 2020 00:56:33 +0100
Subject: Increase timeouts in imfile-basic-2GB-file and
 imfile-truncate-2GB-file

Those tests can take a long time, especially on slow architectures like
armhf, so bump the test timeouts considerably.
---
 tests/imfile-basic-2GB-file.sh    | 4 ++--
 tests/imfile-truncate-2GB-file.sh | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/imfile-basic-2GB-file.sh b/tests/imfile-basic-2GB-file.sh
index d6b40d1..2fa7b9f 100755
--- a/tests/imfile-basic-2GB-file.sh
+++ b/tests/imfile-basic-2GB-file.sh
@@ -5,7 +5,7 @@
 # adds a couple of messages to get it over 2GiB.
 # This is part of the rsyslog testbench, licensed under ASL 2.0
 . ${srcdir:=.}/diag.sh init
-export TB_TEST_MAX_RUNTIME=3600 # test is very slow as it works on large files
+export TB_TEST_MAX_RUNTIME=8000 # test is very slow as it works on large files
 generate_conf
 add_conf '
 module(load="../plugins/imfile/.libs/imfile")
@@ -22,7 +22,7 @@ startup
 ls -lh $RSYSLOG_DYNNAME.input
 export NUMMESSAGES="$(cat $RSYSLOG_DYNNAME.msgcnt)"
 
-wait_file_lines --delay 2500 --abort-on-oversize "$RSYSLOG_OUT_LOG" $NUMMESSAGES 3000
+wait_file_lines --delay 2500 --abort-on-oversize "$RSYSLOG_OUT_LOG" $NUMMESSAGES 6000
 
 # add one message --> exactly 2GB
 ./inputfilegen -m1 -d47 -i$NUMMESSAGES>> $RSYSLOG_DYNNAME.input
diff --git a/tests/imfile-truncate-2GB-file.sh b/tests/imfile-truncate-2GB-file.sh
index 67c9ac1..df45813 100755
--- a/tests/imfile-truncate-2GB-file.sh
+++ b/tests/imfile-truncate-2GB-file.sh
@@ -5,7 +5,7 @@
 # adds a couple of messages to get it over 2GiB.
 # This is part of the rsyslog testbench, licensed under ASL 2.0
 . ${srcdir:=.}/diag.sh init
-export TB_TEST_MAX_RUNTIME=3600 # test is very slow as it works on large files
+export TB_TEST_MAX_RUNTIME=8000 # test is very slow as it works on large files
 generate_conf
 add_conf '
 module(load="../plugins/imfile/.libs/imfile")
@@ -22,7 +22,7 @@ startup
 ls -lh $RSYSLOG_DYNNAME.input
 export NUMMESSAGES="$(cat $RSYSLOG_DYNNAME.msgcnt)"
 
-wait_file_lines --delay 2500 --abort-on-oversize "$RSYSLOG_OUT_LOG" $NUMMESSAGES 3000
+wait_file_lines --delay 2500 --abort-on-oversize "$RSYSLOG_OUT_LOG" $NUMMESSAGES 6000
 
 # add one message --> exactly 2GB
 ./inputfilegen -m1 -d47 -i$NUMMESSAGES>> $RSYSLOG_DYNNAME.input
